server/testutils/factories/cas1PremisesSummary.ts

```diff
@@ -14,5 +14,6 @@ export default Factory.define<Cas1PremisesSummary>(() => ({
   outOfServiceBeds: faker.number.int({ min: 0, max: 50 }),
   apArea: apAreaFactory.build(),
   supportsSpaceBookings: true,
+  managerDetails: `${faker.word.adjective()} ${faker.word.adverb()} ${faker.word.noun()}`,
```

Comment:
Suggest `faker.person` provides a bunch of faked names.

Reply:
thanks for that @bobmeredith - useful - all done!